src/overlay_16_022014A0.c

```diff
@@ -360,7 +372,7 @@ u16 GetTotalBerryQuantity(Bag *bag, HeapID heapId) {
 u16 GetTotalMulchQuantity(Bag *bag, HeapID heapId) {
     s32 i;
     u16 total;
-    for (total = 0, i = 0; i < 4; i++) {
+    for (total = 0, i = 0; i < NUM_MULCHES; i++) {
```

Comment:
`NUM_` to refer to the number of elements in a set is weird, because you could also have `numMulches` which stores the current amount of mulches. TODO investigate what naming we want.